src/platform_impl/macos/window_delegate.rs

```diff
@@ -793,7 +792,7 @@ impl WindowDelegate {
             let size = NSSize::new(logical_size.width, logical_size.height);
             window.setContentSize(size);
         }
-        app_delegate.handle_window_event(window.id(), WindowEvent::Resized(physical_size));
+        self.queue_event(WindowEvent::Resized(physical_size));
```

Comment:
This event was explicitly not queued, since they are executed inside `handle_scale_factor_changed` which is itself queued.

Though I guess that's probably a bit too subtle, so let's just take the miniscule perf hit of redundantly checking whether we need to queue again.